AutomaticInterface/AutomaticInterface/AutomaticInterface.csproj

```diff
@@ -25,7 +25,7 @@
         <PackageLicenseExpression>MIT</PackageLicenseExpression>
         <EnableNETAnalyzers>True</EnableNETAnalyzers>
         <AnalysisLevel>latest-Recommended</AnalysisLevel>
-        <Version>2.3.0</Version>
+        <Version>3.0.0</Version>
```

Comment:
Yes, definitely needed.
I would suggest also adding examples, changelog and your name to Readme.md - especially calling out that you fixed a bug where classes without access modifier where wrongly set to public instead of internal.

Reply:
Good point!